tests/device_tests/test_msg_signtx_amount_unit.py

```diff
@@ -0,0 +1,71 @@
+# This file is part of the Trezor project.
+#
+# Copyright (C) 2012-2020 SatoshiLabs and contributors
+#
+# This library is free software: you can redistribute it and/or modify
+# it under the terms of the GNU Lesser General Public License version 3
+# as published by the Free Software Foundation.
+#
+# This library is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU Lesser General Public License for more details.
+#
+# You should have received a copy of the License along with this library.
+# If not, see <https://www.gnu.org/licenses/lgpl-3.0.html>.
+
+import pytest
+
+from trezorlib import btc, messages as proto
```

Comment:
please don't use `as proto` anymore

Reply:
Fix in b0646ecd69fe29ac8ebec7dcc04b6779a06e0781